docs/schema/okta.md

```diff
@@ -25,7 +25,7 @@
 
 ## OktaOrganization
 
-Representation of a Okta [Organization](https://developer.okta.com/docs/concepts/okta-organizations/).
+Representation of a [Okta Organization](https://developer.okta.com/docs/concepts/okta-organizations/).
```

Comment:
While we're updating the docs, we might as well make the grammar correct :)

Can you do us a huge favor and change 'a' to 'an'? :)

Reply:
Haha yes I can definitely do that :)